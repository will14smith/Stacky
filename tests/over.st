main () -> () { false 0 { over not } invoke print print print }