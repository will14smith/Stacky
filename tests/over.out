=== Output ===
True
0
False

=== Exit ===
0
